bentoml/_internal/runner/runner_handle/remote.py

```diff
@@ -217,7 +217,10 @@ def run_method(
         )
 
     async def is_ready(self) -> bool:
-        async with self._client.get(f"{self._addr}/readyz") as resp:
+        import aiohttp
+
+        timeout = aiohttp.ClientTimeout(total=5)
```

Comment:
This is 5s, not 5m?

Reply:
Thanks for reviewing, Aaron. I did mean to write 5s as I recall Seoyeon saying that the timeout doesn't need to be long, but I could be wrong...@sauyon Could you confirm?

Reply:
I think timeout should fit with k8s timeout @sauyon ?

Reply:
Right. Then I think we should be good to go since k8s performs a liveness probe every 5 seconds per [their docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/).